GliaWidgets/Coordinator/RootCoordinator.swift

```diff
@@ -20,6 +20,7 @@ class RootCoordinator: SubFlowCoordinator, FlowCoordinator {
     private let interactor: Interactor
     private let viewFactory: ViewFactory
     private weak var gliaDelegate: GliaDelegate?
+    private weak var gliaDataSource: GliaDataSource?
```

Comment:
Why is it called data source? As far as I see, it's the window scene which is an UI element, so it doesn't deal with data.

Reply:
Not a good naming indeed, renamed to SceneProvider.